src/views/SessionView/Whiteboard.vue

```diff
@@ -310,6 +309,9 @@ export default {
       // Set brush tool to default tool
       this.useBrushTool();
 
+      // Zoom to full width
+      this.zwibblerCtx.setViewRectangle({ x: 0, y: 0, width: 1000, height: 1 });
```

Comment:
The view needs to be set again once connected to the Zwibbler collab server?

Reply:
Yeah, if you remove this then on re-connection (returning to session from dashboard) it'll zoom to the wrong view